build.assets/Makefile

```
@@ -554,3 +562,74 @@ build-centos7-assets:
 		--build-arg DEVTOOLSET=$(DEVTOOLSET) \
 		--tag $(BUILDBOX_CENTOS7_ASSETS)-$(RUNTIME_ARCH) \
 		-f Dockerfile-centos7-assets .
+
+#################################################################################
+# Multiarch build steps
+#################################################################################
+
+GCC_VERSION ?= 10.4.0
+CLANG_10_VERSION ?= 10.0.1
+CLANG_7_VERSION ?= 7.0.1
```

Can we rename these variables to suggest usage rather than version?
```suggestion
CLANG_BPF_VERSION ?= 10.0.1
CLANG_FIPS_VERSION ?= 7.0.1
```